src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/AuthUserDTO.kt

```diff
@@ -5,12 +5,14 @@ import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUse
 class AuthUserDTO(
```

Comment:
Should this be a data class?

Reply:
yes good spot - saw you handled this so not changing here.